docs/ballerina-by-example/examples/lambda/lambda.description

```diff
@@ -1,6 +1,2 @@
-//Lambdas are simply a syntactic shortcut for defining a function inline.
-//In comparison to a normal function definition, the only missing part here
-//is the name. Semantically, these inline anonymous functions will not be
-//allowed to have any references to in-scope variables other than to global
-//or package level variables. Thus, it behaves exactly like a regular function
-//except that it cannot be called in another place as it has no name.
\ No newline at end of file
+//Simply said, Lambdas are a syntactic shortcut for defining inline functions.
+//In comparison to a normal function definition, the only missing part is the name.
```

Comment:
Can we check this with some developers please? The statement "you can't pass to other places, you can't call from other functions" etc. are missing. Those are needed.

Reply:
Checked with Shavindri. She has already talked to HasithaA. We'll merge.

Reply:
So all that additional info in that comment has been documented elsewhere?

Reply:
With the closure, we allow in-scope variables. 

Regarding "it behaves exactly like a regular function except that it cannot be called in another place as it has no name" This is doesn't make sense now, because you can call it using a function pointer.